instagram/db/schema.rb

```diff
@@ -0,0 +1,29 @@
+# encoding: UTF-8
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
```

Comment:
Metrics/LineLength: Line is too long. [81/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)